common-crafting.lic

```diff
@@ -106,9 +106,17 @@ module DRCC
   def get_crafting_item(name, bag, bag_items, belt)
     waitrt?
     if belt && belt['items'].find { |item| /\b#{name}/i =~ item || /\b#{item}/i =~ name }
-      if 'untie what' == DRC.bput("untie my #{name} from my #{belt['name']}", 'you remove', 'untie what')
-        echo("You seem to be missing: #{name} from your #{belt['name']}")
-        exit
+      case DRC.bput("untie my #{name} from my #{belt['name']}", "You remove", "Untie what")
+	    when "You remove"
+		    return
+      when "Untie what"
```

Comment:
These two cases are very similar. What do you think about removing the `else` on line 121 and letting it handle the 'get' logic (instead of it being part of this `case`)?